dev-docs/bidders/truereach.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: bidder
+title: TrueReach
+description: Prebid TrueReach Bidder Adapter
+biddercode: truereach
+media_types: banner
+---
+
+### Note:
+For more information, please visit us at: [TrueReach Ads](http://doc.truereach.co.in/docs/prebid/js-bidder-adapter.html).
```

Comment:
The landing page for this URL isn't very helpful. Are you sure this is the right URL? Also, it would be better if you provided a secure URL.

Reply:
It is not really a landing page, it is just the "more info" regarding integration that will have our company's contact details and landing page(secure URL). We are already working on the content and UX of "more info" page, please check again.
If secure URL is not a requirement for this page, can we go with un-secure URL for now?